recipes/pspincalc/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "PSpincalc" %}
+{% set version = "0.2.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b8b5f36e72f90ea7b494d63248117f62b0895a019a955ece13f52a22b65b09d2
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - numpy
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - numpy
+    - PSpincalc
+
+about:
+  home: https://github.com/tuxcell/PSpincalc
+  license: GNU General Public v3 or later (GPLv3+)
+  license_family: GPL3
+  license_file: 
```

Comment:
Could you please add the ``license_file``? You can package the license manually
Please see:
https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

Reply:
Hi, yes, of course, it is done now. Thank you so much for your help! :)

Reply:
OK, now it seems to be complete.